backend/devices/bitbox02/keystore_simulator_test.go

```diff
@@ -61,13 +64,16 @@ import (
 	"github.com/btcsuite/btcd/chaincfg"
 	"github.com/btcsuite/btcd/chaincfg/chainhash"
 	"github.com/btcsuite/btcd/wire"
+	"github.com/ethereum/go-ethereum/params"
 	"github.com/stretchr/testify/require"
 )
 
 var (
 	log     = logging.Get().WithGroup("simulator tx signing test")
 	network = &chaincfg.MainNetParams
 	coin    = btc.NewCoin(coinpkg.CodeBTC, "Bitcoin", "BTC", coinpkg.BtcUnitDefault, network, ".", []*config.ServerInfo{}, "https://blockstream.info/testnet/tx/", socksproxy.NewSocksProxy(false, ""))
```

Comment:
[nit] WDYT about renaming this to `coinBTC` as well? to keep consistency with the two new ones

Reply:
Good idea, done!